src/loki/status_action.cc

```diff
@@ -1,8 +1,46 @@
+#include <boost/property_tree/json_parser.hpp>
+#include <iostream>
+#include <sstream>
+
+#include "baldr/tilehierarchy.h"
 #include "loki/worker.h"
+#include "proto/status.pb.h"
 
 namespace valhalla {
 namespace loki {
-void loki_worker_t::status(Api&) const {
+void loki_worker_t::status(Api& request) const {
+
+  auto* status = request.mutable_status();
+
+  std::stringstream ss;
+  boost::property_tree::json_parser::write_json(ss, config);
+  status->set_config(ss.str());
```

Comment:
I also wonder of this is a security risk that should be disablable via config since it tells config details. And also very unfortunate its ptree here because ptree serialize a everything as string iirc. Nothing specific to do just philosophical musings

Reply:
yeah it does serialize everything to string unfortunately..

Reply:
this one is really bothering me... what do you think about the security risk posed by allowing returning the config directly? maybe we need to:

1. put everything behind the verbose flag, to allow status to be just an endpoint that returns 200 when the service is up
2. add a "service limit" for the status "service" that disables the ability to ask for verbose output

in this way someone who is very security conscientious could keep using `status` for health checks but not open up potentially sensitive information (im not really sure if this is true) to regular service users

Reply:
> Nothing specific to do just philosophical musings

I took that quite literal ;) well there's still a few paths for logs and maybe even internal connection ports exposed, which could be used for smth nasty.

ok fair enough. tbh, I wouldn't even have included the config, so for me it's also an option to simply not return it at all. if you wanna keep it, we can do what you suggest as well, your call.

Reply:
hm. yeah forget it, I'll do it anyways, I like just having a health endpoint not doing anything, we could use that for docker health statuses as well.

Reply:
done @kevinkreiser !